### PR TITLE
Fix measure inner dimensions

### PR DESCRIPTION
**Pull request by kinarobin**
`YGFloatIsUndefined(collectedFlexItemsValues.totalFlexGrowFactors) &&            collectedFlexItemsValues.totalFlexGrowFactors == 0`  is not reachable.

### CODE REVIEW

File: yoga/Yoga.cpp [hunk 1]
```diff
@@ -2953,9 +2953,9 @@ static void YGNodelayoutImpl(
       } else {
         if (!node->getConfig()->useLegacyStretchBehaviour &&
             ((YGFloatIsUndefined(
-                  collectedFlexItemsValues.totalFlexGrowFactors) &&
+                  collectedFlexItemsValues.totalFlexGrowFactors) ||
               collectedFlexItemsValues.totalFlexGrowFactors == 0) ||
-             (YGFloatIsUndefined(node->resolveFlexGrow()) &&
+             (YGFloatIsUndefined(node->resolveFlexGrow()) ||
```
Comment (rshest) [inline yoga/Yoga.cpp:2958]: Guess those extra parentheses are redundant, now that it's a list of OR clauses... granted, the expression is quite hard to unpack visually, and extra parentheses don't help here, quite an opposite

Comment (rshest) [reply yoga/Yoga.cpp:2958]: I actually believe that the initial intent there may have been to have it like this instead:
```
if (!node->getConfig()->useLegacyStretchBehaviour &&
            ((!YGFloatIsUndefined(
                  collectedFlexItemsValues.totalFlexGrowFactors) &&
              collectedFlexItemsValues.totalFlexGrowFactors == 0) ||
             (!YGFloatIsUndefined(node->resolveFlexGrow()) &&
              node->resolveFlexGrow() == 0))) {
```

**Review by rshest — CHANGES_REQUESTED** [review]
Please check my comment, the code was wrong there, but I believe the intent was to check if *not* undefined and then compare with zero for both values.

File: yoga/Yoga.cpp (hunk 1 above)
Comment (kinarobin) [reply yoga/Yoga.cpp]: Yeah, you're right. I had update this code.

**Pull request closed without merging**